### ISSUE TITLE
TinyMCE4 popups and label positions are fixed (some browsers)

### ISSUE THREAD

**Issue by bossloper** [evolution-cms/evolution#685]
When using TinyMCE4 toolbar popups (menus/icon titles) seem to be in a fixed position. So ok initially but if you scroll the browser window down then the popups do not re-position to the new location of the toolbar. 

Tested on Chrome and Firefox which both show same issue. 
Windows Edge seems ok (if you do not scroll browser while popup is open).
Also same if using frontend QM+

See e.g.s below
![untitled2](https://user-images.githubusercontent.com/1744687/39970772-4c084726-56e9-11e8-9210-7ef29b32e4ce.png)
![untitled](https://user-images.githubusercontent.com/1744687/39970805-cf1fc648-56e9-11e8-951d-505293fcf5d3.png)

**Comment by bossloper** [comment]
This might be relevant: https://github.com/tinymce/tinymce/issues/3592

**Comment by 64j** [comment]
Maybe there is a problem with this?
https://github.com/evolution-cms/evolution/commit/454f4f975988a9c1e87c93237c9c8295ff65be54

**Comment by bossloper** [comment]
@64j Yes I think you are right :-( https://github.com/davidjbradshaw/iframe-resizer/issues/83 
But what to do?! Without the iOS fixes it is bad on iPhone/iPad. I do not know of an alternative to fix iOS?

**Comment by 64j** [comment]
@bossloper 
Return it as it was before.
Add styles for ios-only classes

Here is the class for ios ( .ios ...... )
https://github.com/evolution-cms/evolution/blob/develop/manager/frames/1.php#L134

**Comment by bossloper** [comment]
It is not just classes though, there is a 'scrolling="no"' on the iframe.

**Comment by 64j** [comment]
@bossloper 
```css
.ios iframe { overflow: hidden !important }
```
Iframe remove attribute "scrolling"

**Comment by bossloper** [comment]
@64j - using overflow: hidden to change scrolling does not work. 
I added .ios classed and all working good except for the scrolling="auto" / scrolling="no", which if I test by manually changing on the iframe tag it works but using css 'overflow: hidden' technique does not give the desired result on an iOS device.

**Comment by 64j** [comment]
@bossloper 
Unfortunately, I can not check.
I do not have ios

**Comment by bossloper** [comment]
OK - Thanks @64j 
I do now think I have a solution to only apply the alternative iframe scrolling solution for ios. It does use javascript to set the scrolling=no but it seems to work. I will get the code together as a commit so it can be tested...

**Comment by bossloper** [comment]
https://github.com/evolution-cms/evolution/pull/686

**Comment by kkumar981** [comment]
Hi.

I want to show popover on selection of element. Editor is in read only mode. can you please help me out how to show popup on text selection inside iframe.

Thanks,
Kumar
